test/unit_test/cb_las_one_pass_svd_test.cc

```diff
@@ -58,21 +58,21 @@ BOOST_AUTO_TEST_CASE(check_AO_same_actions_same_representation)
       VW::multi_ex examples;
 
       examples.push_back(VW::read_example(vw, "shared |U b c"));
```

Comment:
adding namespaces in these examples would make the tests fail without these changes